blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/inputs/handlers/HttpHandlerIntegrationTest.java

```diff
@@ -134,18 +92,17 @@ public void testHttpIngestionInvalidPastCollectionTime() throws Exception {
 
     @Test
     public void testHttpIngestionInvalidFutureCollectionTime() throws Exception {
+
+        String prefix = getPrefix();
+
         long time = System.currentTimeMillis() + 600000 + Configuration.getInstance().getLongProperty( CoreConfig.AFTER_CURRENT_COLLECTIONTIME_MS );
 
-        HttpPost post = new HttpPost(getMetricsURI());
-        HttpEntity entity = new StringEntity(JSONMetricsContainerTest.generateJSONMetricsData( time ),
-                ContentType.APPLICATION_JSON);
-        post.setEntity(entity);
-        HttpResponse response = client.execute(post);
+        HttpResponse response = postGenMetric( TID, prefix, postPath, time );
 
         assertEquals( 400, response.getStatusLine().getStatusCode() );
-
         String[] output = getBodyArray( response );
 
+        assertEquals( 4, output.length );
```

Comment:
the first line is boilerplate, "The following errors have been encountered:"  I can add that to the tests if you'd like.